src/client.js

```diff
@@ -171,10 +171,14 @@ assign(Client.prototype, {
   },
 
   wrapIdentifier(value) {
+    return this.customWrapIdentifier(value, this.wrapIdentifierImpl);
+  },
+
+  customWrapIdentifier(value, origImpl) {
```

Comment:
Kind of hacky, but works at least as long as user doesn not implement their custom quote style too in their wrapIdentifier configuration. Maybe knex would need this approach to be more official and to have separate APIs to override for quoting and for transforming identifiers. Anyways this is good enough for now.

Reply:
Yeah, but there is no other way to do this right now.